constructorio-client/src/test/java/io/constructor/client/ConstructorItemTest.java

```diff
@@ -19,29 +19,31 @@ public String getRandomProductName() {
     public ExpectedException thrown = ExpectedException.none();
 
     @Test
-    public void newWithNullItemNameShouldFail() throws Exception {
+    public void newWithNullItemNameShouldPass() throws Exception {
```

Comment:
Should this be `newWithNullItemIdShouldPass`?

Reply:
It should have been `newWithNullItemIdShouldFail`.
Great catch!